splunk-otel-android/src/main/java/io/opentelemetry/rum/internal/instrumentation/activity/ActivityTracer.java

```diff
@@ -14,21 +14,27 @@
  * limitations under the License.
  */
 
-package com.splunk.rum;
+package io.opentelemetry.rum.internal.instrumentation.activity;
+
+import static io.opentelemetry.rum.internal.RumConstants.APP_START_SPAN_NAME;
+import static io.opentelemetry.rum.internal.RumConstants.SCREEN_NAME_KEY;
+import static io.opentelemetry.rum.internal.RumConstants.START_TYPE_KEY;
 
 import android.app.Activity;
 import androidx.annotation.NonNull;
 import androidx.annotation.Nullable;
+import com.splunk.rum.RumScreenName;
```

Comment:
Oh, I totally forgot about this annotation. I think we'll have to introduce some `(Fragment|Activity)NameExtractor` in a future PR/story that'll help us decouple this.

Reply:
Ah sheesh, I forgot about it too. We'll have to do another sweep of the remaining `splunk` stuff in there anyway (there were others last I checked).